README.md

```diff
@@ -14,3 +11,32 @@
 [![Imports: isort](https://img.shields.io/badge/%20imports-isort-%231674b1?style=flat&labelColor=2e3a44)](https://pycqa.github.io/isort/)
 [![Checked with mypy](http://www.mypy-lang.org/static/mypy_badge.svg)](http://mypy-lang.org/)
 [![Code Style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg)](https://github.com/prettier/prettier)
+
+
+_A collection of packages used to process imagery_
+
+# Packages
```

Comment:
As we have only one package for now (and probably want only one for this repository for simplicity), we won't have a list of different containers.
What do you think of removing the `s` of `Packages` and not having a list?